### PR TITLE
TVR-9723 Update Request v4 API documentation

### PR DESCRIPTION
**Pull request by laurentlhermine**
Add Request Type in Header definition
Add sorting field and sorting order parameters for the list of Requests
Update format of custom fields in examples

### CODE REVIEW

File: src/api-reference/request/v4.endpoints.markdown [hunk 1]
```diff
@@ -147,6 +154,12 @@ None
 
 ## <a name="request-resource-update"></a>Update the content of an existing Request
 
+Update of the the following fields is supported : policy, comment, startDate, startTime, endDate, endTime, expensePolicy, name, businessPurpose, mainDestination, travelAgency, custom fields.<br>
+Other fields will be ignored.<br>
+If providing an "id" field, as returned by GET requests, the value must match the requestUuid parameter.<br>
+This endpoint supports partial update. You may submit only the fields to update in the body, fields not present in the body will remain unchanged.<br>
```
Comment (sarraloew) [inline src/api-reference/request/v4.endpoints.markdown]: This update contradicts line 174 below- which one is correct?

File: src/api-reference/request/v4.endpoints.markdown [hunk 2]
```diff
@@ -525,6 +540,7 @@ totalPostedAmount|object|[Amount](#amount)|The total amount of the Request, expr
 totalRemainingAmount|object|[Amount](#amount)|The total amount not included in an Expense report, expressed in the reimbursement currency of the employee at the time he created the Request
 travelAgency|object|[ResourceLink](#schema-resourcelink)|The travel agency office that is managing the trip associated to this Request
 operations|array|[Link](#schema-link)|Links to operations available for the Request, depends on the current approval status
+type|[RequestType](#schema-requesttype)|The type of the Request, inherited from the Request Policy Type
```
Comment (sarraloew) [inline src/api-reference/request/v4.endpoints.markdown]: This table row is missing a column.

**Review by sarraloew — CHANGES_REQUESTED** [review]

File: src/api-reference/request/v4.endpoints.markdown (hunk 1 above)
Comment (laurentlhermine) [reply src/api-reference/request/v4.endpoints.markdown]: Updated in new commit

File: src/api-reference/request/v4.endpoints.markdown (hunk 2 above)
Comment (laurentlhermine) [reply src/api-reference/request/v4.endpoints.markdown]: Updated in new commit

**Review by sarraloew — APPROVED** [review]

**Comment by sarraloew** [comment]
:shipit:

**Pull request merged**